dnstable_convert.c

```diff
@@ -1027,17 +1024,18 @@ usage(const char *name)
 			"<NMSG FILE> <DB FILE> <DB DNSSEC FILE>\n", name);
 
 	fprintf(stderr, "Options:\n"
-	" -b SIZE:  The uncompressed DNS (i.e. not DNSSEC) data block size hint for the output file..\n"
-	" -c TYPE:  Use TYPE compression (Default: zlib)\n"
-	" -D:       Put CDS, CDNSKEY, and TA RRSets in both outputs\n"
-	" -l LEVEL: Use numeric LEVEL of compression.\n"
-	"           Default varies based on TYPE.\n"
-	" -m MMB:   Specify maximum amount of memory to use for in-memory sorting, in megabytes.\n"
-	" -p:       Preserve empty DNS/DNSSEC files.\n"
-	" -r:       Emit RDATA and RDATA_NAME_REV dnstable entries for SOA rname field.\n"
-	" -s NAME:  NMSG source information to include in output if input is stdin.\n"
-	" -S:       Include nmsg source information in output.\n"
-	" -t COUNT: Use a maximum of COUNT threads during sorting and writing.\n");
+		" -b size:  set mtbl block size to use for DB FILE (default: %u).\n"
```

Comment:
This indentation and lower-casing is incorrect.

The metavar capitalization (e.g., `-b SIZE`) is a long-standing convention, and we should follow. Similarly, DNS rrtype mnemonics (`CDS, CDNSKEY, and TA`) are conventionally capitalized. Also, the `-D` option is upper case, and appears lowercase in the help text.

Reply:
This is my fault. I am not sure what happened here... but it seems obvious that I accidentally lower-cased everything in vim when making the indent, and somehow missed those changes.

Reply:
Pushed